manim_slides/present.py

```diff
@@ -688,6 +720,14 @@ def update_image(self, cv_img: np.ndarray) -> None:
 
         self.label.setPixmap(QPixmap.fromImage(qt_img))
 
+    @Slot()
+    def update_canvas(self) -> None:
+        """Update the canvas when a presentation has changed."""
+        logger.debug("Updating canvas")
+        self.display_width, self.display_height = self.thread.current_resolution
+        if not self.isFullScreen():
+            self.resize(self.display_width, self.display_height)
```

Comment:
@Fairlight8 Third, every time the presentation changes, call to `resize`.
There, you will need to call `setStyleSheet(...)`.